### PR TITLE
docs: Document host replacement

### PR DESCRIPTION
**Pull request by josephglanville**
Documents use of promote/demote to replace Raft consensus members.

### CODE REVIEW

File: docs/content/operating.html.md [hunk 1]
```diff
@@ -249,3 +249,21 @@ all hosts. The installed version of Flynn can be checked with `flynn-host
 version`, and the version to install can be specified by setting the
 `FLYNN_VERSION` environment variable to the desired version when running the
 install script.
+
+# Replacing Hosts
+
+If an initial member of the cluster that is participating in the consensus set becomes permenantly unavailable it must be replaced in order to restore fault tolerance.
```
Comment (titanous) [inline docs/content/operating.html.md]: permanently

File: docs/content/operating.html.md [hunk 2]
```diff
@@ -249,3 +249,21 @@ all hosts. The installed version of Flynn can be checked with `flynn-host
...
+If you have already added additional hosts to your cluster beyond the initial bootstrap set then you can simply promote one of these peers to the consensus set.
```
Comment (titanous) [inline docs/content/operating.html.md]: drop simply

File: docs/content/operating.html.md [hunk 3]
```diff
@@ -249,3 +249,21 @@ all hosts. The installed version of Flynn can be checked with `flynn-host
...
+Otherwise you will first need to start a new host and join it to the cluster as describred in the Adding Hosts section.
```
Comment (titanous) [inline docs/content/operating.html.md]: described

File: docs/content/operating.html.md [hunk 4]
```diff
@@ -249,3 +249,21 @@ all hosts. The installed version of Flynn can be checked with `flynn-host
...
+
+From this point you must first demote the old peer from the consensus set.
+If the peer is already unavailable the use of the `--force` flag is necessary, if however you are premtively replacing the peer it can be ommited for a more graceful stepdown.
```
Comment (titanous) [inline docs/content/operating.html.md]: preemptively

Comment (titanous) [inline docs/content/operating.html.md]: omitted

File: docs/content/operating.html.md [hunk 5]
```diff
@@ -249,3 +249,21 @@ all hosts. The installed version of Flynn can be checked with `flynn-host
...
+The command to do so is `flynn-host demote --force $PEER_IP`.
+
+Once the old peer is removed we can add the new peer to the consensus set with `flynn-host promote $PEER_IP`.
```
Comment (titanous) [inline docs/content/operating.html.md]: s/we/you

**Comment by josephglanville** [comment]
I think we need to better explain where Raft is in-use and how that impacts importance of specific nodes a bit better. Maybe we also need to enhance either `flynn-host list` or add another command to show which peers are currently Raft members.
Not remembering peers is currently somewhat of a limitation here, `flynn-host` isn't really aware that other hosts exists except by proxy of using `discoverd`.
More thoughts on how to make this better are definitely required.

**Comment by josephglanville** [comment]
@titanous ready for final review.

File: docs/content/operating.html.md [hunk 6]
```diff
@@ -249,3 +249,22 @@ all hosts. The installed version of Flynn can be checked with `flynn-host
 version`, and the version to install can be specified by setting the
 `FLYNN_VERSION` environment variable to the desired version when running the
 install script.
+
+# Replacing Hosts
+
+If a member of the cluster that is participating in the consensus set becomes permanently unavailable it must be replaced in order to restore fault tolerance.
+If you have already added additional hosts to your cluster beyond the members of the consensus set then you can promote one of these peers to the consensus set.
+Otherwise you will first need to start a new host and join it to the cluster as described in the Adding Hosts section.
+You can check the Raft consensus status of your cluster hosts by running `flynn-host list`, if the host is a member it's status will be displayed as `peer`, conversely it will be listed as `proxy`.
```
Comment (titanous) [inline docs/content/operating.html.md]: its

**Comment by titanous** [comment]
LGTM

**Pull request merged**